app/editor/src/components/form/wysiwyg/CustomToolbar.tsx

```diff
@@ -48,7 +50,7 @@ export const CustomToolbar: React.FC<ICustomToolbarProps> = ({
       <button className="ql-image"></button>
     </span>
     <span className="ql-formats">
-      <button>
+      <button type="button" onClick={onClickExpand}>
```

Comment:
We use the `<Button/>` component for consistency and the ability to change them all easily.  Don't use the base button HTML element.

Reply:
It seems the base button element has been used everywhere in this file. Do you think if we should have another ticket to replace the base element throughout the app? Thanks

Reply:
This is specific to the WYSIWYG toolbar - which from the docs is most compatible with the base html buttons! I am sure we can make it work with <Button /> but as far as WYSIWYG styling goes this may be best